tests/specs/packages.js

```diff
@@ -0,0 +1,192 @@
+define(["dist/amd-loader"], function(AMDLoader) {
+  describe("Test suite", function() {
+    var bitloader;
+    beforeEach(function() {
+      bitloader = new AMDLoader({
+        baseUrl: "../",
+        packages: [
+          {
+            location: "tests",
+            main: "main",
+            name: "js"
+          }
+        ]
+      });
+    });
+
+    describe("When Importing a number", function() {
+      var number;
+      beforeEach(function() {
+        return bitloader.import("js/number.js")
+          .then(function (_number) {
+            console.log(arguments);
+            number = _number;
+          });
+      });
+
+      it("then the number is 3.14", function() {
+        expect(number).to.equal(3.14);
+      });
+    });
+
+    describe("When Importing an empty object", function() {
+      var result;
+      beforeEach(function() {
+        return bitloader.import("js/emptyobject.js")
+          .then(function (_result) {
+            result = _result;
+          });
+      });
+
+      it("then the result is an object", function() {
+        expect(result).to.be.an("object");
+      });
+
+      it("then the result is an object with no properties", function() {
+        expect(Object.keys(result).length).to.equal(0);
+      });
+    });
+
+    describe("When Importing an empty object and an empty string", function() {
+      var emptyobject, emptystring;
+      before(function () {
+        return bitloader.import(["js/emptyobject", "js/emptystring"])
+          .then(function (_emptyobject, _emptystring) {
+            emptyobject = _emptyobject;
+            emptystring = _emptystring;
+          });
+      });
+
+      describe("and the empty object is loaded", function () {
+        it("then `emptyobject` is an object", function () {
+          expect(emptyobject).to.be.an("object");
+        });
+        it("then `emptyobject` does not have any properties", function() {
+          expect(Object.keys(emptyobject).length).to.equal(0);
+        });
+      });
+
+      describe("and the empty string is loaded", function () {
+        it("then `emptystring` is a string", function() {
+          expect(emptystring).to.be.a("string");
+        });
+        it("then `emptystring` has length 0", function() {
+          expect(emptystring.length).to.equal(0);
+        });
+      });
+    });
+
+    describe("When Importing nested dependencies in deep3", function() {
+      var deep3;
+      beforeEach(function () {
```

Comment:
We can do a `before` here and get rid of the `describe("and deep3 is loaded", function() {});`